CHANGELOG.md

```diff
@@ -6,13 +6,46 @@
 
 ## History
 
+- [v1.5.0](#v150)
 - [v1.4.1](#v141)
 - [v1.4.0](#v140)
 - [v1.3.0](#v130)
 - [v1.2.0](#v120)
 - [v1.1.0](#v110)
 - [v1.0.0](#v100)
 
+## v1.5.0
+
+Learn more about our release in [our milestone](https://github.com/kedacore/keda/milestone/12).
```

Comment:
I've bundled them in a release, maybe a good idea for doing this in the future in a `vNext` milestone and rename when we know the version?

Most of the time was figuring out what we are shipping.